pkg/schemaconv/openapi.go

```diff
@@ -25,6 +25,12 @@ import (
 	"sigs.k8s.io/structured-merge-diff/v4/schema"
 )
 
+// Expects a map from definition name to OpenAPI V3 structural schema for each definition.
```

Comment:
Shouldn't the comments start with `// ToSchemaFromOpenAPI ...`

Reply:
Does this accept an openapi v2? How are we supposed to use it for built-ins? Do we generate a structural schemas for these? That sounds like a waste.

Reply:
Builtins are all OpenAPIV3. Usually they are same schema for v2 and v3. The builtins which need a separate v2 and v3 schema provide one. (There is a hack which stuffs the v2 schema inside the v3schema as an extension)

Reply:
added // ToSchemaFromOpenAPI ...